Products/GenericSetup/registry.py

```diff
@@ -552,12 +559,11 @@ def listForbiddenTools(self):
     def addForbiddenTool(self, tool_id):
         """ See IToolsetRegistry.
         """
+        self._required.pop(tool_id, None)
+
```

Comment:
Why not use the same pattern as in `addRequiredTool` below (pop without using the value just seems awkward).